include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -2140,7 +2125,7 @@ namespace internal
         const DoFCellAccessor<dim, spacedim, level_dof_access> &accessor)
       {
         if (accessor.dof_handler->hp_capability_enabled == false)
-          return 0; // ::DoFHandler only supports a single active FE with index
+          return 0; // DoFHandler only supports a single active FE with index
```

Comment:
These comments will be removed in #14866.

Reply:
I'll undo these edits - thanks for catching that.